utilities/data/getStoryblokRedirects.mjs

```diff
@@ -100,12 +100,13 @@ export const getStoryblokRedirects = async () => {
     return [];
   }
 
-  const redirects = stories.map(entry => ({
+  const redirects = stories
+  .filter(entry => entry.content.from && entry.content.to) // Filter out entries without "from" or "to"
+  .map(entry => ({
     source: sanitizeSourcePath(entry.content.from),
     has: extractQueryParameters(entry.content.from),
     destination: entry.content.to,
-    statusCode: sanitizeRedirectCode(entry.content.statusCode),
+    statusCode: sanitizeRedirectCode(entry.content.statusCode || 301),
```

Comment:
You shouldn't need the `|| 301` here as the sanitize function should default to that already
https://github.com/SU-SWS/ood-giving-campaign/blob/003d5de75eae1ad8d58438e373728b65489d7dc5/utilities/data/getStoryblokRedirects.mjs#L43